### PR TITLE
convert tests to jest

### PR DESCRIPTION
**Pull request by garrensmith**
Convert all mocha/phantomjs tests to jest.

### CODE REVIEW

File: app/addons/documents/doc-editor/__tests__/doc-editor.components.test.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      }
+    });
+
+    const $attachmentNode = el.find('.view-attachments-section .dropdown-menu li');
+    const attachmentURLactual = $attachmentNode.find('a').first().prop('href');
+
+    assert.equal(attachmentURLactual, '../../id/_design/test-doc/one.png');
+  });
+
+  it.skip('setting deleteDocModal=true in store shows modal', () => {
```
Comment (Antonio-Maranhao) [inline app/addons/documents/doc-editor/__tests__/doc-editor.components.test.js]: Test passes if you remove `skip`.

File: app/addons/documents/doc-editor/__tests__/doc-editor.components.test.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    // no modal is open, then confirms the open modal contains the delete dialog message
+    assert.equal($('body').find('.confirmation-modal').length, 0);
+
+    Actions.showDeleteDocModal();
+
+    const modalContent = $('body').find('.confirmation-modal .modal-body p')[0];
+    assert.ok(/Are you sure you want to delete this document\?/.test(modalContent.innerHTML));
+  });
+
+  it.skip('setting uploadDocModal=true in store shows modal', () => {
```
Comment (Antonio-Maranhao) [inline app/addons/documents/doc-editor/__tests__/doc-editor.components.test.js]: Test passes if you remove `skip`.

File: app/addons/setup/__tests__/setupComponents.test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import utils from "../../../../test/mocha/testUtils";
+import React from "react";
+import ReactDOM from "react-dom";
+import sinon from "sinon";
+import { mount } from 'enzyme';
+
+const assert = utils.assert;
+
+//this was commented out. I imagine it needs to be updated
+describe.skip('Setup Components', () => {
```
Comment (Antonio-Maranhao) [inline app/addons/setup/__tests__/setupComponents.test.js]: The tests work fine - although with a couple of warning msgs from React.

**Comment by Antonio-Maranhao** [comment]
+1 Great work @garrensmith 
I tried a few of the skipped tests and they still work if you want to re-enable them.

**Comment by garrensmith** [comment]
@Antonio-Maranhao I'm going to merge this and add back the skipped tests in another PR.

**Pull request merged**